csharp/extractor/Semmle.Extraction.CSharp/Entities/LocalVariable.cs

```diff
@@ -22,12 +22,19 @@ class LocalVariable : CachedSymbol<ISymbol>
 
         public override void Populate()
         {
+            if (symbol is ILocalSymbol local)
+            {
+                ExtractNullability(local.NullableAnnotation);
+                if (local.IsRef)
+                    Context.Emit(Tuples.type_annotation(this, Kinds.TypeAnnotation.Ref));
+            }
+
             Context.Emit(Tuples.localvars(
                 this,
                 IsRef ? 3 : IsConst ? 2 : 1,
```

Comment:
We should add a ticket for simplifying `localvars` now that we record `ref` information in `type_annotation`.

Reply:
https://jira.semmle.com/browse/ODASA-7997